README.md

```diff
@@ -170,12 +170,42 @@ To start the trento agent:
 > recommendations on cluster settings and state for many HA components. New rules
 > can be created and tuned to adjust to different requirements.
 
+## Trento Runner
+
```

Comment:
The section assumes that user is going to run the runner and ARA on the same machine, this is not always true. When I tried to replicate the installation on k3s i realized that my trento runner container was missing the python3 `ara.setup.*` dependencies. Could we split the section in two, one for the ARA setup and one for the trento runner? Also, is installing the `ara[server]` dep needed on the trento runner machine/container, or we can install just the `ara` module without the server deps?

Reply:
I will add new content to make clear that the ARA server can go in another machine.
And I will have a look about the ARA dependencies when the 2 components are running in different locations